src/test/kotlin/stringAddCalculator/CalculatorTest.kt

```diff
@@ -9,7 +9,7 @@ class CalculatorTest : FreeSpec({
 
     "calculate" - {
 
-        "정상적인 게산기 동작 확인" {
+        "조건에 맞는 입력값에 따라 덧셈이 되는 동작 확인" {
```

Comment:
```suggestion
        "입력값이 유효한 경우 덧셈이 되는 동작 확인" {
```
"조건에 맞는" 이라는 의미도 정확히 어떤 조건인지에 대해서 알 수 없기에, "유효하다"의 의미를 담으면 좋을 것 같아요.